hw/mcu/nordic/src/ext/nrfx/mdk/nrf52_bitfields.h

```diff
@@ -11169,6 +11169,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #define TWI_FREQUENCY_FREQUENCY_Msk (0xFFFFFFFFUL << TWI_FREQUENCY_FREQUENCY_Pos) /*!< Bit mask of FREQUENCY field. */
 #define TWI_FREQUENCY_FREQUENCY_K100 (0x01980000UL) /*!< 100 kbps */
 #define TWI_FREQUENCY_FREQUENCY_K250 (0x04000000UL) /*!< 250 kbps */
+#define TWI_FREQUENCY_FREQUENCY_K380 (0x06147ae9UL) /*!< 380 kbps */
```

Comment:
@anangl should we add this in nrfx?

Reply:
No. Such value is not mentioned in the Product Specification, so it's not officially supported. Thus, we can't add it to nrfx (MDK actually).

Reply:
No, I don't think it's worth the effort. It's as feature, not a bug. Can we just merge this? Next time a NRFX update takes place we need to do better review.

Reply:
I think we should add this to i2c_nrf52_twim.c, rather than maintain a custom patched NRFX library.  I don't see usage of this anywhere but in that driver, is that correct?

Reply:
we can just use hardcoded (+comment) value in both hal_i2c and i2c_nrf52_twim instead of using symbolic name